Source/WebCore/page/DragController.cpp

```diff
@@ -1210,7 +1210,7 @@ bool DragController::startDrag(Frame& src, const DragState& state, OptionSet<Dra
 #if ENABLE(ATTACHMENT_ELEMENT)
     if (is<HTMLAttachmentElement>(element) && m_dragSourceAction.contains(DragSourceAction::Attachment)) {
         auto& attachment = downcast<HTMLAttachmentElement>(element);
-        auto* attachmentRenderer = attachment.renderer();
+        auto* attachmentRenderer = dynamicDowncast<RenderAttachment>(attachment.renderer());
```

Comment:
Since we're changing this code, would you mind moving this down closer to where it's used?

(Underneath the `if (!dragImage)`)